packages/react/src/lib/factories.ts

```diff
@@ -46,70 +46,101 @@ const mappedProps: { [key in HTMLTag]: ShorthandProp } = {
 // ============================================================
 
 /** A more robust React.createElement. It can create elements from primitive values. */
-export function createShorthand(
-  Component: React.ReactType,
-  mappedProp?: string,
-  valueOrRenderCallback?: ShorthandValue | ShorthandRenderCallback,
-  options: CreateShorthandOptions = CREATE_SHORTHAND_DEFAULT_OPTIONS,
-): React.ReactElement<Props> | null | undefined {
+export function createShorthand({
+  Component,
+  mappedProp,
+  mappedArrayProp,
+  valueOrRenderCallback,
+  options: optionsPram,
+}: {
+  Component: React.ReactType
+  mappedProp?: string
+  mappedArrayProp?: string
+  valueOrRenderCallback?: ShorthandValue | ShorthandRenderCallback
```

Comment:
Not sure about this name here... Would rather have value, as it is much simpler and shorter...